### ISSUE TITLE
(phi0660.phi003) correct URN assignment?

### ISSUE THREAD

**Issue by lcerrato** [PerseusDL/canonical-latinLit#256]
These are Sulpicia texts but this looks like a Tibullus URN. Any ideas on this @AlisonBabeu?

**Comment by AlisonBabeu** [comment]
Hi @lcerrato. I think this ID decision was made a long time ago when there weren't STOA IDs around, including one for this Sulpicia, who does not have a PHI ID. This issue initially confused me greatly when I first started cataloging Perseus editions because there is a PHI ID for a different Sulpicia [phi1023](http://catalog.perseus.org/catalog/urn:cite:perseus:author.1344) but there is a STOA ID, [stoa0269](http://catalog.perseus.org/catalog/urn:cite:perseus:author.1343) for this Sulpicia in question.  I think they used this ID because the Sulpicia text is part of the Corpus Tibulllianum, and the only actual Tibullus PHI IDs are phi0660.phi001 (Elegiae) and phi0660.phi002 (Carmina Tibullianum sp.)

I actually associated this work ID phi0660.phi003 to the correct Sulpicia, and cataloged the Perseus edition under it (http://catalog.perseus.org/catalog/urn:cts:latinLit:phi0660.phi003) because since it had already been published under that ID I wasn't sure what other approach to take. I'm not sure if this is at all helpful.

**Comment by lcerrato** [comment]
Hi @AlisonBabeu 
The catalog totally confused me because http://catalog.perseus.org/catalog/urn:cts:latinLit:phi0660.phi003.perseus-lat1 has no indication of Sulpicia anywhere. This might be a display issue, but all of the phi0660 pages only reference Tibullus.

Likewise, when I looked here:
http://catalog.perseus.org/catalog/urn:cite:perseus:author.1343
I just started looking for works with stoa0269 (I'm looking in the data itself so I didn't scroll down).

I have no problem assigning a better URN and moving the texts to that but I understand that our practice to this point is not to do this as it severs any permanency. 

This also creates the problem of it not being findable in the Scaife Viewer right now. If you type in Sulpicia, you won't access these works. I could correct that with the cts author/work files, but that would be messy (I guess I would have to list two different authors?)

**Comment by AlisonBabeu** [comment]
Yeah I know the current display options with Blacklight could never represent this properly  unfortunately.  This is where you have to click on the Atom feed to get the whole MODS record where it shows Sulpicia listed as the "attributed author", but the current catalog implementation can only associate/show one author/editor with any given MODS record. 

You certainly can move the files and assign a different URN, I guess it would be stoa0269.stoa002.perseus-lat1. In terms of the cts file, I think you would need to list two authors, but I'm not entirely certain on that. Happy to do more investigating tomorrow!

**Comment by lcerrato** [comment]
Either the author has to be Corpus Tibulllianum, which no one will ever find, or they should be split. I don't think we want the two author solution in the cts files if we can avoid it. 
More tomorrow.
